frontend/src/utils/profanity.ts

```diff
@@ -0,0 +1,5 @@
+const Filter = require('bad-words');
```

Comment:
Looks good! Tested it and it works. Nit: if we want to keep the import style consistent, you could do what [this post ](https://stackoverflow.com/a/50516783)says and create a `.d.ts` file to get rid of the ts module declaration error.